alexandria/core/serializers.py

```diff
@@ -324,3 +311,20 @@ class Meta:
             "content",
         )
         extra_kwargs = {"content": {"write_only": True}}
+
+
+class WebDAVSerializer(BaseSerializer):
+    webdav_url = serializers.SerializerMethodField()
+
+    def get_webdav_url(self, instance):
+        if not settings.ALEXANDRIA_USE_MANABI:
+            return None
```

Comment:
Should we issue a 400 or 404 here? Or is the frontend determining if manabi is used only by receiving `null` here?

Reply:
IMO we should throw a 404. Also, could you please add a setting e.g. `ALEXANDRIA_MANABI_ALLOWED_MIMETYPES` with default to docx and xlsx and throw a 404 if the mimetype of the file doesn't match?

I discussed this with @czosel and we both agree that we need such an option because of security reasons of various customers. We don't want to expose all files to webdav.

Reply:
Raises 404 in `check_permission` when not enabled